website/docs/rbs-support.md

```diff
@@ -261,6 +261,16 @@ attr_writer :bar
 attr_accessor :baz
 ```
 
+Long attribute accessors signatures can span over multiple lines:
```

Comment:
```suggestion
Long signatures can span over multiple lines:
```

Reply:
We already have a section earlier for method signatures, this one is about attr accessor types.

I replaced `attribute accessors signatures` by `attribute accessors types` 👍